src/lib/GraphDataUtils.js

```
@@ -0,0 +1,698 @@
+
+    case 'average':
+      return values.reduce((acc, val) => acc + val, 0) / values.length;
+
+    case 'difference':
+      // Subtract all subsequent values from the first
+      return values.reduce((acc, val, index) => index === 0 ? val : acc - val, 0);
+
+    case 'ratio':
+      // Divide first value by second (or product of all others)
+      if (values.length < 2) {return null;}
+      const denominator = values.slice(1).reduce((acc, val) => acc * val, 1);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)